Composer/cypress/support/commands.ts

```diff
@@ -24,6 +24,15 @@ Cypress.Commands.add('visitPage', page => {
   cy.findByTestId('ActiveLeftNavItem').should('contain', page);
 });
 
+Cypress.Commands.add('enterTextAndSubmit', (textElement: string, text: string, submitBtn?: string) => {
+  cy.findByTestId(textElement)
+    .clear()
+    .type(text);
+  if (submitBtn) {
+    cy.findByTestId(submitBtn).click();
+  }
+});
```

Comment:
I see. the reasoning behind it was actually get it to a review and see how u feel about it. Basically, it had its utility in a few places and I thought this could be a nice utility to type text and submit. But, u have a lot more expertise on this project and if you feel it does not add value i can remove it and let enter submit the components

Reply:
I don't think it abstracts away enough logic to warrant a new method. Also the name and its body are in conflict as it doesn't always submit, only if you pass in a test id to the submit button.